pulpcore/pulpcore/app/serializers/base.py

```diff
@@ -97,7 +97,7 @@ class GenericKeyValueRelatedField(serializers.DictField):
 
     You can store anything you want in here, as long as it's a string.
     """
-    child = serializers.CharField()
+    child = serializers.CharField(allow_blank=True)
```

Comment:
allow_blank should not be set here. 
the child field is a drf construct to validate items in a DictField: http://www.django-rest-framework.org/api-guide/fields/#dictfield